feeds/src/config.ts

```diff
@@ -24,3 +24,63 @@ export interface OracleNode {
   name: string
   networkId: number
 }
+
+class UrlConfig {
+  static feedsJson(location: Location): string | undefined {
+    const query = new URLSearchParams(location.search)
+    const feedsJson = query.get('feeds-json')
+
+    if (feedsJson) {
+      return decodeURIComponent(feedsJson)
+    }
+    return undefined
+  }
+
+  static nodesJson(location: Location): string | undefined {
+    const query = new URLSearchParams(location.search)
+    const nodesJson = query.get('nodes-json')
+
+    if (nodesJson) {
+      return decodeURIComponent(nodesJson)
+    }
+    return undefined
+  }
+}
+
+export class Config {
+  static infuraKey(env = process.env): string {
+    if (env.REACT_APP_INFURA_KEY === undefined) {
+      return ''
+    }
+    return env.REACT_APP_INFURA_KEY
```

Comment:
```
return env.REACT_APP_INFURA_KEY ?? ''
```

Reply:
Any reason why we return an empty string instead of undefined?

Reply:
Main reason was to make it a single return type. But don't feel super strongly about it

Reply:
I think its good that we differentiate a missing value from a string